src/phoenix/server/api/routers/v1/datasets.py

```diff
@@ -584,24 +560,16 @@ async def _get_dataset_download_csv(request: Request) -> Response:
             )
         examples = [
             {
-                **metadata,
-                **input,
-                **output,
-                "__example_index__": example_index,
+                "example_id": GlobalID(type_name=DatasetExample.__name__, node_id=str(example_id)),
+                **{f"input_{k}": v for k, v in input.items()},
+                **{f"output_{k}": v for k, v in output.items()},
+                **{f"metadata_{k}": v for k, v in metadata.items()},
```

Comment:
This is changing the keys of the example JSON so that it diverges from what is shown in the UI and stored in the DB. Can we instead return a nested JSON object with an ID and the example unchanged, e.g.,

```
{
    "id": ...,
    "example": {
        "input": ...,
        "output": ...,
        "metadata": ...
    }
}
```

Reply:
Forgot this is CSV, so we wouldn't want nested. I think this makes sense.

Reply:
Although maybe we want to support the original JSON object format since that is the format people will have written their evals for. Not sure.